components/Event.js

```diff
@@ -256,8 +259,7 @@ class Event extends React.Component {
                   <div className="eventDescription">
                     <Markdown source={event.longDescription || event.description} escapeHtml={false} />
                   </div>
-
-                  <section id="tickets">
+                  <section id="tickets" className={isEventOver && event.endsAt ? 'hide' : ''}>
```

Comment:
Why using css `display: none` rather than having a condition to not render the DOM at all? Something like:

```es6
{!event.endsAt || !isEventOver) && (
  <section id="tickets>
    ...
  </section>
)}
```

Reply:
The test was failing, so, I was trying to fix it but I've removed the styling